composer.json

```diff
@@ -71,7 +71,7 @@
     "require": {
         "php":                          ">=5.3.3",
         "symfony/framework-bundle":     "~2.3",
-        "symfony/security-bundle":      "~2.1",
+        "symfony/security-bundle":      "~2.6",
```

Comment:
@stof May be I'm not so skilled for now, but the I see only way to keep compatability is to reimplement that hack with ```security.context```, which already exist in Sf 2.6 (Internally convert security context to managers). I'm not sure, which is more painful to maintain.

If there is a more clever way - give me an idea an I try to implement it.

With these changes I hope make it compatible with 2.6+ and 3.0+